etc/crm.conf.in

```diff
@@ -21,6 +21,9 @@
 ; report_tool_options =
 ; lock_timeout = 120
 
+; set OCF_1_1_SUPPORT to yes is to turn the ocf 1.1 feature once the corresponding CIB detected.
+; OCF_1_1_SUPPORT = no
```

Comment:
```suggestion
; OCF_1_1_SUPPORT = yes
```

Reply:
Don't we set it as `no` by default for now?

Reply:
I assume the code set the default to "no" already. I read the above line are in the comment context. It has no influence to the code at all. And, the user can simply remove ";" to uncomment it to change the default "no" to "yes".